src/middleware/packages/auth/mixins/auth.js

```diff
@@ -63,29 +77,109 @@ const AuthMixin = {
         // TODO make sure token is deleted client-side
         ctx.meta.webId = 'anon';
         return Promise.reject(new E.UnAuthorizedError(E.ERR_INVALID_TOKEN));
+      } else if ((method === 'Capability' || req.$params.capability) && this.settings.podProvider) {
```

Comment:
Why do you want to enable this only on pod providers ? I think it could work well for other SemApps instances, no ?

Reply:
Ah, good point ! In multi-users SemApps instances, we could look at the `dc:creator` property of the capability, and ensure that it is the same as the owner of the resource it gives access to (or that it has `acl:Control` permission on that resource). But we should prevent a user to change the (automatically-created) `dc:creator` property of the capability. In the end, that doesn't seem so secure... Maybe in these cases we should consider signing the capability, like in the [zcap](https://w3c-ccg.github.io/zcap-spec/) proposal. But OK to disable this in non-Pod config for now !